src/asm/lexer.c

```diff
@@ -921,6 +921,32 @@ void lexer_DumpStringExpansions(void)
 	free(stack);
 }
 
+/* Discards an block comment */
+static void discardBlockComment(void)
+{
+	dbgPrint("Discarding block comment\n");
+	for (;;) {
+		switch (nextChar()) {
+		case EOF:
+			error("Unterminated block comment\n");
+			return;
+		case '/':
+			if (nextChar() == '*') {
+				warning(WARNING_NESTED_COMMENT,
+					"/* in block comment\n");
+				continue;
```

Comment:
Actually, this `continue` should be in the other branch: currently, `/* //` ends the comment, but `/* /*/\ doesn't.
Test cases (`weird-block-comment`?) should be added for those, too;